parser-typechecker/src/Unison/Runtime/Rt1.hs

```diff
@@ -211,6 +211,7 @@ data Result
   = RRequest Req
   | RMatchFail Size [Value] Value
   | RDone Value
+  | RError String
```

Comment:
I'd like to make this `RError ErrorType Value`, basically runtime won't be responsible for converting these errors to strings.

```
type ErrorType = Bug | Todo
```